src/pkg/caendr/caendr/services/cloud/utils.py

```diff
@@ -24,6 +24,34 @@
 
 
 
+def make_dns_name_safe(s: str) -> str:
```

Comment:
Abstracted out from `TaskHandler`. It technically doesn't do everything specified by the format -- might have to update this.